sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -98,6 +98,7 @@ private static JavaCommand createWebServerCommand(Props props, File homeDir) {
       .addJavaOptions(props.nonNullValue(ProcessProperties.WEB_JAVA_ADDITIONAL_OPTS))
       // required for logback tomcat valve
       .setEnvVariable(ProcessProperties.PATH_LOGS, props.nonNullValue(ProcessProperties.PATH_LOGS))
+      .setEnvVariable("TMPDIR", props.nonNullValue(ProcessProperties.PATH_TEMP))
```

Comment:
Don't you think that it deserves to mention JRuby in a comment ? It's quite unusual in a pure Java environment as setting the system property java.io.tmpdir is supposed to be enough.